lib/chipmunk/bagger/audio_local_metadata.rb

```diff
@@ -6,9 +6,9 @@
 
 module Chipmunk
   class Bagger::AudioLocalMetadata < Bagger
-    def initialize(external_id:, bag_path:, src_path: nil, metadata_url:, metadata_type:, metadata_path:)
+    def initialize(content_type:, external_id:, bag_path:, src_path: nil, metadata_url:, metadata_type:, metadata_path:)
```

Comment:
This is pretty leaky... But we have to either leak (cede content type responsibility at the Bagger) or duplicate, as in PR #9. I definitely didn't like adding a very duplicative spec over there (it obscures what is unique), but I'm also not very keen on increasing essential complexity and conditionality like this.

I think we have to be practical and choose which might be easier to refactor into something with more composition of behavior. I need to sleep on it.